ops/terraform/services/eft/modules/bfd_eft_outbound_o11y/lambda_src/outbound_slack_notifier/outbound_slack_notifier.py

```diff
@@ -107,6 +107,11 @@ def handler(event: dict[Any, Any], context: LambdaContext):  # pylint: disable=u
             status_notification = StatusNotification(**raw_notification)
 
             logger.append_keys(notification_type=status_notification.type.value)
+         
+            if (status_notification.type in [
+                        NotificationType.TRANSFER_SUCCESS,
+                        NotificationType.FILE_DISCOVERED ]):
+               continue 
```

Comment:
This will need to be formatted using `ruff`.

Install `uv` using `brew install uv`, and then `cd` into this directory in a shell and run:

```
uvx ruff format
```

The above command should re-format this line to conform to existing formatting standards.